Source/Plugins/ImportExport/ImportExportConstants.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Your License should go here
+ */
+#ifndef _importexportconstants_h_
+#define _importexportconstants_h_
+
+#include <QtCore/QString>
+
+/**
+* @brief This namespace is used to define some Constants for the plugin itself.
+*/
+namespace ImportExportConstants
+{
+  const QString ImportExportPluginFile("ImportExportPlugin");
+  const QString ImportExportPluginDisplayName("ImportExport");
+  const QString ImportExportBaseName("ImportExport");
+
+  namespace BlueQuartz
+  {
+    const QString VendorName("BlueQuartz Software, LLC");
+    const QString URL("http://www.bluequartz.net");
+    const QString Copyright("(C) 2018 BlueQuartz Software, LLC");
+  } // namespace BlueQuartz
+
+  namespace FilterGroups
+  {
+  	const QString ImportExportFilters("ImportExport");
```

Comment:
Unless it was updated since the last time I tried it, clang-format would have flattened the constants file to the left-side.  I figured not running it on this file was intentional.